docs/content/1.getting-started/3.theming.md

```diff
@@ -269,4 +269,4 @@ export default defineAppConfig({
     }
   }
 })
-```
+```
```

Comment:
Why is there a change here? Can't see the difference 🤔

Reply:
👀 ooh, solving conflicts

Reply:
@benjamincanac done now 😉